packages/vm/tests/StatelessRunner.js

```diff
@@ -0,0 +1,194 @@
+const ethUtil = require('ethereumjs-util')
+const BN = ethUtil.BN
+const { getRequiredForkConfigAlias, setupPreConditions, makeTx, makeBlockFromEnv } = require('./util')
+const Account = require('@ethereumjs/account').default
+const Trie = require('merkle-patricia-tree').SecureTrie
+const { default: Common } = require('@ethereumjs/common')
+const { default: VM } = require('../dist/index.js')
+const { default: DefaultStateManager } = require('../dist/state/stateManager')
+
+async function runTestCase (options, testData, t) {
+  let expectedPostStateRoot = testData.postStateRoot
+  if (expectedPostStateRoot.substr(0, 2) === '0x') {
+    expectedPostStateRoot = expectedPostStateRoot.substr(2)
+  }
+
+  // Prepare tx and block
+  let tx = makeTx(testData.transaction)
+  let block = makeBlockFromEnv(testData.env)
+  tx._homestead = true
+  tx.enableHomestead = true
+  block.isHomestead = function () {
+    return true
+  }
+  if (!tx.validate()) {
+    return
+  }
+
+  const common = new Common('mainnet', options.forkConfigVM.toLowerCase())
+  const stateManager = new DefaultStateManager({ common: common })
+  await setupPreConditions(stateManager._trie, testData)
+  const preStateRoot = stateManager._trie.root
+
+  // Set up VM
+  let vm = new VM({
+    stateManager: stateManager,
+    common: common
+  })
+  if (options.jsontrace) {
+    hookVM(vm, t)
+  }
+
+  // Determine set of all node hashes in the database
+  // before running the tx.
+  const existingKeys = new Set()
+  const it = stateManager._trie.db.iterator()
```

Comment:
Can you also give me a hint, where this `iterator()` might have moved to? This was errored as not available when I last ran the code, got stock there to some extend.

Reply:
This was possibly a local change, can't find an iterator method in MPT too. Levelup has an `iterator` [method](https://github.com/Level/levelup#dbiteratoroptions) which I probably used. Maybe just replacing that line with `stateManager._trie.db._leveldb.iterator()` would work.